content/plugins/openstack.md

```diff
@@ -17,9 +17,12 @@ For more information about OpenStack, please refer to: [https://www.openstack.or
 
 # Plugin Requirements
 
-* Python Versions:
+* Python versions:
   * 2.7.x
-
+* Additional packages:
```

Comment:
Please use a note saying that if the plugin is installed from source, it requires these additional system dependencies.